### PR TITLE
M24Y-10: Complete Warehouse detail component

### PR DESCRIPTION
**Pull request by tina96pham**
- Add warehouse detail component. Please test by adding this to the end of the localhost route: /warehouse/1 
- Move the jsx code from testing page to WarehousePage
- NOTE: please pay attention to the the font size in the warehosuse-details__text which is the body text for thee info. It is a bit off for desktop, not sure if it is my eyes.
-

### CODE REVIEW

File: src/components/WarehouseDetails/WarehouseDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                {warehouse.contact_phone}
+              </p>
+              <p className="warehouse-details__text">
+                {warehouse.contact_email}
+              </p>
+            </li>
+          </ul>
+        </div>
+      ) : (
+        <div class="page-loading">
```
Comment (kimntan) [inline src/components/WarehouseDetails/WarehouseDetails.jsx:70]: className

File: src/utils/useFetchWarehouse.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {useState, useEffect, useMemo} from "react";
+import {useParams} from "react-router-dom"
+import InstockApi from "./instock-api";
+
+const useFetchWarehouse = () => {
+  const {id} = useParams()
+  const instockApi = useMemo(() => new InstockApi(), []);;
```
Comment (kimntan) [inline src/utils/useFetchWarehouse.js:7]: double semi colon

**Review by kimntan — COMMENTED** [review]
Great job! I like your hook!

File: src/components/WarehouseDetails/WarehouseDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        <div className="warehouse-details">
+          <div className="warehouse-details__subheader">
+            <Link to="/">
+              <img
+                src={arrowBackIcon}
+                alt="Back Icon"
+                className="warehouse-details__back-icon"
+              />
+            </Link>
+            <h2 className="warehouse-details__title">
```
Comment (kimntan) [inline src/components/WarehouseDetails/WarehouseDetails.jsx:22]: This should be an h1

**Comment by crystdang** [comment]
You're right about the font size, the text should be slightly larger than the titles. Also the padding around the text should be slightly smaller which made the text feel smaller

**Pull request merged**